clang/test/dpct/cmake_migration/case_021/input.cmake

```diff
@@ -11,3 +11,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
 add_link_options("--add-arch=sm_70")
 list(APPEND PMEMD_NVCC_FLAGS --std c++11)
+
+string(APPEND CMAKE_C_FLAGS " -prec_div")
+string(APPEND CMAKE_CXX_FLAGS " -parallel")
+string(APPEND CMAKE_CXX_FLAGS " -wd188,186,144,913,556,858,597,177,1292,167,279,592,94,2722,3199")
```

Comment:
1. These three options are not NVCC-specific options,  which compiler uses those options?
2. In my opinion, if it is only used in a specific application for a specific compiler, we can implement those migration rules as application-specific rules?

Reply:
1. These options are used by Intel Classical C Compiler (ICC) - [ref](https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-8/alphabetical-option-list.html)
2. I haven't found any other examples of apps using these flags yet

Reply:
Then,  I suggest that those migration rules should be implemented as application-specific rules.

Reply:
It's kind of migration knowledge, how about put it into a separate cmake_script_migration_rule_optional.yaml?